src/cascadia/TerminalControl/TermControl.h

```diff
@@ -194,6 +197,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         winrt::Windows::UI::Xaml::Controls::SwapChainPanel::LayoutUpdated_revoker _layoutUpdatedRevoker;
 
+        void _UpdateSettingsFromUIThread(IControlSettings newSettings);
+        void _UpdateAppearanceFromUIThread(IControlAppearance newAppearance);

```

Comment:
You removed `_UpdateSettingsOnUIThread()` right? Don't forget to remove it from the .h file.